internal/services/kusto/kusto_cluster_resource_test.go

```diff
@@ -241,6 +241,15 @@ func TestAccKustoCluster_vnet(t *testing.T) {
 			),
 		},
 		data.ImportStep(),
+		{
+			Config: r.vnetRemoved(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("virtual_network_configuration.#").HasValue("0"),
+				check.That(data.ResourceName).Key("public_network_access_enabled").HasValue("false"),
+			),
+		},
+		data.ImportStep(),
```

Comment:
Can we add an additional test step where we add the vNet block back into the config?
```suggestion
		{
			Config: r.vnetRemoved(data),
			Check: acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).ExistsInAzure(r),
				check.That(data.ResourceName).Key("virtual_network_configuration.#").HasValue("0"),
			),
		},
		data.ImportStep(),
		{
			Config: r.vnet(data),
			Check: acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).ExistsInAzure(r),
				check.That(data.ResourceName).Key("virtual_network_configuration.#").HasValue("1"),
				check.That(data.ResourceName).Key("virtual_network_configuration.0.subnet_id").Exists(),
				check.That(data.ResourceName).Key("virtual_network_configuration.0.engine_public_ip_id").Exists(),
				check.That(data.ResourceName).Key("virtual_network_configuration.0.data_management_public_ip_id").Exists(),
			),
		},
		data.ImportStep(),
```